src/grasshopper/lib/grasshopper.py

```diff
@@ -67,8 +68,18 @@ def influx_configuration(self) -> dict[str, Optional[str]]:
         if pwd:
             configuration["pwd"] = pwd
 
+        configuration["ssl"] = self.to_bool(self.global_configuration.get("influx_ssl", "False"))
+        configuration["verify_ssl"] = self.to_bool(self.global_configuration.get("influx_verify_ssl", "False"))
+
         return configuration
 
+    @staticmethod
```

Comment:
As noted above, you don't really need this method. The `typecast_bool` method already exists and is automatically getting applied to all the variables.